reflekt-plugin/src/main/kotlin/io/reflekt/plugin/analysis/ir/ArgumentCollectors.kt

```diff
@@ -29,16 +30,15 @@ class ReflektInvokeArgumentsCollector : IrRecursiveVisitor() {
         super.visitCall(expression, data)
         val function = expression.symbol.owner
         when (function.name.asString()) {
-            ReflektFunction.WITH_SUPERTYPE.functionName -> {
-                supertypes.addAll(expression.getFqNamesOfTypeArguments())
-            }
-            ReflektFunction.WITH_SUPERTYPES.functionName -> {
-                supertypes.addAll(expression.getFqNamesOfClassReferenceValueArguments())
-            }
+            ReflektFunction.WITH_SUPERTYPE.functionName -> supertypes.addAll(expression.getFqNamesOfTypeArguments())
+            ReflektFunction.WITH_SUPERTYPES.functionName -> supertypes.addAll(expression.getFqNamesOfClassReferenceValueArguments())
             ReflektFunction.WITH_ANNOTATIONS.functionName -> {
                 annotations.addAll(expression.getFqNamesOfClassReferenceValueArguments())
                 supertypes.addAll(expression.getFqNamesOfTypeArguments())
             }
+            else -> {
```

Comment:
Are you sure about using these empty blocks? I mean it could complicate the source code. Maybe it is better to disable this inspection?

Reply:
Sure, that can be suppressed. That was automatically generated to attract your attention.
I have seen several bugs realted to a missing else block in the code, but I also do not like [this inspection](https://github.com/diktat-static-analysis/diKTat/blob/master/info/guide/diktat-coding-convention.md#-311-when-expression)